### DISCUSSION TITLE
Link query help

### DISCUSSION THREAD

**Discussion by dwrinn** [mdelobelle/metadatamenu#619] [Q&A]
Hi, I am trying my to write a basic query where I am want to see only the files with the tag Project and I not sure how to do it. 
I currently have dv.pages('"#Project"') and I am still getting all of the files in my vault. What am I doing wrong?

**Comment by mdelobelle** [comment]
try `dv.pages("#Project")` (without the surrounding single quotes)

**Reply by dwrinn** [reply]
No I tried both just single and double quotes
